webapp/src/pages/offer/components/OfferComponent.js

```diff
@@ -129,51 +128,49 @@ class Offer extends Component {
          { this.state.showReasonBox ? 
           <div class="form-group mr-5  ml-5 pt-5" >
           <textarea class="form-control pr-5 pl-10" id="textArea" onChange={()=>{this.handleChange(this.rejected_reason)}}  placeholder="Enter rejection message"></textarea>
-          <button type="button" class="btn" id="apply" onClick={
-            ()=>{
+          <button type="button" class="btn" id="apply" onClick={ ()=>{
               this.setState({
                 showReasonBox:false
               },
               this.buttonSubmit());
-            }
-            }>apply</button>
+            }}>apply</button>
         </div>
-        :""}
+        :""
+        }
         </div>
       
       </form>
-  </div>
+    </div>
       )}
 
-    componentWillMount(){
+    componentDidMount(){
       userService.get().then(result=> {
         this.setState({
             user:{
-              user_id:result.user_id
+              user_id:result.response.data.app_user_id
```

Comment:
where is this `response` variable?
Why are we setting a `user_id` object? 
Look at how this was done. Look at authenticate, look at the user service. 

I'm going to have to finish this since we're just not getting there